environment.md

```diff
@@ -1,19 +1,35 @@
-# Environment
+# Server Environment
 
-WordPress will run in a minimum server environment. However, WordPress doesn't run optimally on the minimum system requirements. This section will cover the _recommended_ server environment for WordPress to run more effectively.
+Although WordPress can work in almost any environment, even very minimal ones, it must be acknowledged that it does not work completely well in these. That's why here we are going to make some minimum recommendations of the environment in which it would work most effectively.
 
-## Recommended Servers
+## Web Server
 
-WordPress recommends using one of these servers:
+The web server is the system where the files of the web site are hosted and where the users arrive to consult them. There are many web servers, and in principle anyone who supports a PHP connection should be able to work with WordPress.
 
-*   Apache 2.4 or higher
-*   NGINX 1.14 or higher
+When it comes to the server, web, WordPress works best with these (sorted alphabetically):
+
+* [Apache HTTPD](https://httpd.apache.org/) 2.4.x
+* [LiteSpeed Web Server](https://www.litespeedtech.com/products/litespeed-web-server) 5.3.x / 5.4.x
+* [nginx](https://nginx.org/) 1.18.x / 1.17x
++ [OpenLiteSpeed](https://openlitespeed.org/) 1.7.x / 1.6.x / 1.5.x / 1.4.x
+
+Remember that if you have a website running in production, it is recommended to use the latest stable version of each of the web servers (mainly for security, rather than functionality), but not alpha, beta or candidate (RC) versions.
 
 ## PHP
 
-### PHP Version
+PHP is a programming language on which WordPress code is based. This language runs on the server and it is important to keep it up to date, both for security and mainly for functionality.
+
+WordPress supports many versions of PHP, some already obsolete, but always all that are supported and maintained.
+
+Officially the WordPress core supports from PHP 7.0 to PHP 7.4. However, not all themes or plugins are supported.
+
+When it comes to PHP, WordPress (including its extensions) works best with the following versions:
 
-PHP `7.3` or greater is highly encouraged but WordPress will run on older versions. Please note, however, that versions of PHP lower than `7.2` have reached end-of-life status (EOL) and are no longer receiving security updates. For this reason, PHP versions lower than `7.2` are not recommended.
+* [PHP 7.4](https://www.php.net/ChangeLog-7.php#PHP_7_4).x
+* [PHP 7.3](https://www.php.net/ChangeLog-7.php#PHP_7_3).x
+* [PHP 7.2](https://www.php.net/ChangeLog-7.php#PHP_7_2).x
+
+WordPress does not work with versions less than 7.0. Versions prior to PHP 7.2 are not recommended because it no longer has support of any kind, and only PHP 7.2 if you have the latest version, since it only has security support.
```

Comment:
WordPress works with PHP 5.6.20+ (from https://wordpress.org/about/requirements/)

Is there a reason you linked to the PHP changelogs for the major versions? I'm wondering if something a little simpler that talks about ranges would be be easier to update, and still have the necessary information.

Reply:
Changed to:

_WordPress does with PHP 5.6+. Versions prior to PHP 7.2 are not recommended because it no longer has support of any kind, and only PHP 7.2 if you have the latest version, since it only has security support._

About the links, is to show the updates so anyone can check security and functionality. I can link to prior versions.

Reply:
Okay, sounds good.

Maybe something like "WordPress works with 5.6.20+, but versions before 7.2 are not recommended due to lack of support from the PHP project. It's always recommended to use the latest security release of whatever version is installed."?

5.6.20+, specifically, is important to note, because it doesn't work with versions of 5.6 earlier than 5.6.20 (meaning 5.6 doesn't work, for instance).

Reply:
fixing it.fixed